src/shared/sam/cli/samCliValidator.ts

```diff
@@ -10,7 +10,7 @@ import { SamCliInfoInvocation, SamCliInfoResponse } from './samCliInfo'
 import { SamCliProcessInvoker } from './samCliInvokerUtils'
 
 export const MINIMUM_SAM_CLI_VERSION_INCLUSIVE = '0.16.0'
-export const MAXIMUM_SAM_CLI_VERSION_EXCLUSIVE = '0.30.0'
+export const MAXIMUM_SAM_CLI_VERSION_EXCLUSIVE = '0.40.0'
```

Comment:
It's broken without the other change though so it's actually worse to increment. It's not that big of a deal since we are doing the other change right after but it is a worse state.

Reply:
yup, changed for testing but accidentally commited